python/lsst/ctrl/bps/wms/panda/panda_service.py

```diff
@@ -198,38 +201,38 @@ def from_generic_workflow(cls, config, generic_workflow, out_prefix, service_cla
         idds_workflow = cls(generic_workflow.name, config)
         workflow_generator = IDDSWorkflowGenerator(generic_workflow, config)
         idds_workflow.generated_tasks = workflow_generator.define_tasks()
-        cloud_prefix = config['bucket'] + '/' + \
+        file_placement_path = config['fileDistributionEndPoint'] + '/' + \
             config['payload_folder'] + '/' + config['workflowName'] + '/'
-        cls.copy_pickles_into_cloud([config['bps_defined']['run_qgraph_file']], cloud_prefix)
+        cls.copy_files_for_distribution([config['bps_defined']['run_qgraph_file']], file_placement_path)
```

Comment:
Which files need to be copied should be determined from the GenericWorkflow itself and not hardcoding a specific one from the config.